tests/acceptance/features/bootstrap/WebUIAdminGeneralSettingsContext.php

```diff
@@ -156,6 +156,49 @@ public function theAdministratorSetsTheLogLevelUsingTheWebui($logLevel) {
 		$this->adminGeneralSettingsPage->setLogLevel($logLevel);
 	}
 
+	/**
+	 * @When the administrator adds group :lockBreakerGroup to the lock breakers groups using the webUI
+	 *
+	 * @param string $lockBreakerGroup
+	 *
+	 * @return void
+	 */
+	public function theAdministratorAddsGroupToLockBreakersGroupUsingTheWebui($lockBreakerGroup) {
+		$this->adminGeneralSettingsPage-> addGroupLockBreakersGroup(
+			$this->getSession(),
+			$lockBreakerGroup
+		);
+	}
+
+	/**
+	 * @Then group :expectedGroup should be listed in the lock breakers groups in the webUI
+	 *
+	 * @param $expectedGroup
+	 *
+	 * @return void
+	 */
+	public function groupShouldBeListedAsLockBreakersGroupInTheWebui($expectedGroup) {
+		$actualGroup = $this->adminGeneralSettingsPage-> getLockBreakersGroups();
+		if (!\in_array($expectedGroup, $actualGroup)) {
+			Assert::assertFalse(
+				"$expectedGroup should be present in lock breakers groups, but it isn't"
+			);
+		}
+	}
+
+	/**
+	 * @Then following groups should be listed in the lock breakers groups in the webUI
```

Comment:
```suggestion
	 * @Then the following groups should be listed in the lock breakers groups on the webUI
```

We usually say "on the webUI" in the step text.